config/initializers/database.rb

```diff
@@ -3,7 +3,10 @@
 
   ActiveSupport.on_load(:active_record) do
     config = Rails.application.config.database_configuration[Rails.env]
-    config['pool'] = (ENV['DB_POOL'] || 100).to_i
-    ActiveRecord::Base.establish_connection(config)
+
+    if config
+      config['pool'] = (ENV['DB_POOL'] || 100).to_i
+      ActiveRecord::Base.establish_connection(config)
+    end
```

Comment:
What are the scenarios where we'd want to run the database initializer but not attempt to connect to the database?